src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseDataTest.java

```diff
@@ -617,11 +617,11 @@ private DwpDocument buildDwpDocument(String documentUrl, DwpDocumentType documen
     private ScannedDocument buildScannedDocument(String documentUrl, DocumentType documentType, String date, String controlNumber) {
         String docType = documentType == null ? null : documentType.getValue();
         return ScannedDocument.builder().value(
-                ScannedDocumentDetails.builder().type(docType)
-                        .url(DocumentLink.builder().documentUrl(documentUrl).build())
-                        .scannedDate(date)
-                        .controlNumber(controlNumber)
-                        .build()).build();
+            ScannedDocumentDetails.builder().type(docType)
```

Comment:
minor: formatting - can we new line the in-line builders as well? can show you what I mean on a call if you want.